scripts/o3de/ExportScripts/export_standalone_monolithic_engine_centric.py

```diff
@@ -38,16 +38,25 @@
 
 # For more information on the available parameters for this script, check the parse_arguments function defined at the bottom of the file
 
+
+# Account for some windows-specific attributes
+if platform.system().lower()=='windows':
+    EXECUTABLE_EXTENSION = '.exe'
+    ADDITION_BUILD_ARGS = ['--', '/m', '/ nologo']
```

Comment:
There's a space here - also the `/m` is dangerous on Threadrippers right?  I'm actually wonder if we should have a way for the user to control the additional build args.  One way is to use CMake build presets and let the user optionally pass in a build preset that they can customize in a `CMakeUserPresets.json` or use our existing preset which would contain the args above.  The downside is it means the user needs to learn about CMake presets instead of just adding the args in Python.  Thoughts?
```suggestion
    ADDITION_BUILD_ARGS = ['--', '/m', '/nologo']
```

Reply:
Actually we should make the additional build arguments and additional configure arguments an argparse option.

Also the `/m` and `/nologo` options aren't required for building on Windows.`

Reply:
I suggested that we also just look for a `--` argument and pass all the arguments after that down to cmake. However, the nature of this script doesn't make it a natural choice because there are a lot of processes that occur:

1. cmake project generation
2. cmake build
3. asset bundler
4. zip process

This particular fix was to fix and error that occurs on Linux. I'll let @tkothadev decide how to handle the `/m` arguments

Reply:
If CMake customization proves to be ornery, it may be best to create a separate script for enabling that customization. The current scripts are meant to capture common use cases (80% of export experience) and be simple enough for others to adapt as needed. Moving some functions into the export_project API library may be an appropriate step here.